docs/src/routes/button/index.svelte

```diff
@@ -5,3 +5,10 @@
 <script>
   import Doc from "./demo.md";
 </script>
+
+<style global>
```

Comment:
Instead of defining this here, it may be better to put these in each demo so that in the future when someone wants to copy the code to try it out or if we send them to a site to demo like  codesandbox.io they can have the spacing.

Reply:
I can not do that because the in the actual example we don't use `ant-button` as a class and svelte will remove it automatically
<img width="699" alt="Screenshot 2020-04-14 at 7 11 10 PM" src="https://user-images.githubusercontent.com/14311743/79253635-d48c4700-7e83-11ea-9ba8-9edfdcf4e604.png">

I think it is fine for now to keep it like this.

Reply:
Sounds good